sdk/servicebus/Microsoft.Azure.WebJobs.Extensions.ServiceBus/src/Config/ServiceBusExtensionConfigProvider.cs

```diff
@@ -74,14 +74,6 @@ public void Initialize(ExtensionConfigContext context)
                 throw new ArgumentNullException(nameof(context));
             }
 
-            // Set the default exception handler for background exceptions
-            // if not already set.
-            Options.ProcessErrorAsync ??= (e) =>
```

Comment:
Why do we move this from Initialize?

Reply:
This was moved to `ServiceBusTriggerAttributeBindingProvider` as we need to access the function name. But I just realized that in T1 we clone `ServiceBusOptions` and `ServiceBusOptions `is different for each function. For T2 ServiceBusOptions is the same and it will report only the function name which was on create.

I do not see how it can be done without cloning..

Reply:
I see.. I'm a little unsure about this change in general because we are changing the category for existing events.

If we think this change is important, we'd probably need to implement it by not mutating ProcessErrorAsync value on the options and instead pass in the handler as a new argument to ServiceBusListener constructor. Given that the event category already has ServiceBusListener in the name, and it matches what we do for all other events (using the type name from where it was logged), I'd suggest we hold off on this change until we get feedback that it is necessary. If I'm not mistaken in T1 we went from having a category of just Executor to function name, whereas in T2 we already have the extension-specific type.